community_images/node-exporter/bitnami/dc_coverage.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -x
+set -e
+# shellcheck disable=SC1091
+SCRIPTPATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+
+# shellcheck disable=SC1091
+. "${SCRIPTPATH}"/../../common/scripts/bash_helper.sh
+
+JSON_PARAMS="$1"
+
+JSON=$(cat "$JSON_PARAMS")
+
+echo "Json params for docker compose coverage = $JSON"
+
+PROJECT_NAME=$(jq -r '.project_name' < "$JSON_PARAMS")
+
+CONTAINER_NAME="${PROJECT_NAME}"-node-exporter-1
+docker inspect "${CONTAINER_NAME}"
+sleep 5
+
+docker exec "${CONTAINER_NAME}" /opt/bitnami/node-exporter/bin/node_exporter --version
+
+# Check the metrics using cURL for node-exporter
+curl http://localhost:9100/metrics
```

Comment:
Can we use dynamic port mapping here on?
Refer: apache/official

Reply:
Updated